src/mixins/itext_key_behavior.mixin.ts

```diff
@@ -111,7 +111,7 @@ export abstract class ITextKeyBehaviorMixin<
    * Override this method to customize cursor behavior on textbox blur
    */
   blur() {
-    this.abortCursorAnimation();
+    !this.selected && this.abortCursorAnimation();
```

Comment:
not sure what `this.selected` is anymore and if is consistent across all fabric.
I would rather not use it.

Reply:
let's not make this change. if the user select a ui textarea the default is to not show 2 cursor animations

Reply:
exactly
I was aiming for that and I am to blame for it, see #7784
The thing is it causes a flicker on mouse down since the hidden textarea blurs and immediately focuses.

Reply:
But maybe best to have this is a separate PR and test it with all cases...

Reply:
`selected` is an internal thing in text, maybe should be removed